VSCode/armsnippets.json

```diff
@@ -758,6 +758,213 @@
+      "            \"name\": \"[concat(${1:account-name}, '/sql/', ${2:database-name})]\",",
+      "            \"apiVersion\": \"2016-03-31\",",
+      "            \"dependsOn\": [ \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', ${1:account-name})]\" ],",
+      "            \"properties\":{",
+      "                \"resource\":{",
+      "                    \"id\": \"${2:database-name}\"",
+      "                },",
+      "                \"options\": { \"throughput\": \"${3:throughput}\" }",
+      "            }",
+      "        }"
+    ],
+    "description": "Cosmos DB SQL Database"
+  },
+
+  "Cosmos DB Mongo Database": {
+    "prefix": "arm-cosmos-mongo-database",
+    "body": [
+      "      {",
+      "            \"type\": \"Microsoft.DocumentDB/databaseAccounts/apis/databases\",",
+      "            \"name\": \"[concat(${1:account-name}, '/mongodb/', ${2:database-name},'/',${3:throughput})]\",",
+      "            \"apiVersion\": \"2016-03-31\",",
+      "            \"dependsOn\": [ \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', ${1:account-name})]\" ],",
+      "            \"properties\":{",
+      "                \"resource\":{",
+      "                    \"throughput\": \"${3:throughput}\"",
+      "                },",
+      "            }",
+      "        }"
+    ],
+    "description": "Cosmos DB SQL Database"
+  },
+  "Cosmos DB Gremlin Database": {
+    "prefix": "arm-cosmos-gremlin-database",
+    "body": [
+      "        {",
+      "            \"type\": \"Microsoft.DocumentDB/databaseAccounts/apis/databases\",",
+      "            \"name\": \"[concat(${1:account-name}, '/gremlin/', ${2:database-name})]\",",
+      "            \"apiVersion\": \"2016-03-31\",",
+      "            \"dependsOn\": [",
+      "                \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', ${1:account-name})]\"",
+      "            ],",
+      "            \"properties\": {",
+      "                \"resource\": {",
+      "                    \"id\": \"${2:database-name}\"",
+      "                }",
+      "                \"options\": {",
+      "                    \"throughput\": \"${3:throughput}\"",
+      "                }",
+      "            }",
+      "        },"
+    ],
+    "description": "Cosmos DB Gremlin Database"
+  },
+  "Cosmos DB Cassandra Namespace": {
+    "prefix": "arm-cosmos-cassandra-namespace",
+    "body": [
+      "{",
+      "			\"type\": \"Microsoft.DocumentDB/databaseAccounts/apis/keyspaces\",",
+      "			\"name\": \"[concat(${1:account-name}, '/cassandra/', ${2:database-name})]\",",
+      "			\"apiVersion\": \"2016-03-31\",",
+      "			\"dependsOn\": [ \"[resourceId('Microsoft.DocumentDB/databaseAccounts/', ${1:account-name})]\" ],",
+      "			\"properties\":{",
+      "				\"resource\":{",
+      "					\"id\": \"${3:keyspace}\"",
+      "				},",
+      "				\"options\": { \"throughput\": \"${4:throughput}\" }",
+      "			}",
+      "		},"
+    ],
```

Comment:
4-space indentation please

Reply:
This should be resolved now, sorry for the delay